source/develop/release-management/features/virt/virt-v2v-integration.html.md

```diff
@@ -139,6 +139,33 @@ Import from external environments using virt-v2v in terms of the general flow:
 
 10. Unlock VM & disks
 
+##### Import OVA from VMware
+
+Currently oVirt support importing OVA only from VMware hypervisor.
+The OVA formats that oVirt supported are tar, zip and folder.
```

Comment:
Well it is supported now and we had a bug to support it.
When we have the upload ova we can specify what we support in the wiki page of the upload